tuna/miopen/parse_miopen_args.py

```diff
@@ -85,62 +91,29 @@ def get_import_cfg_parser(
                       type=str,
                       default=None,
                       help='Specify driver cmd')
-  parser.add_argument('-f',
-                      '--file_name',
-                      type=str,
-                      dest='file_name',
-                      help='File to import')
-  parser.add_argument('-F',
-                      '--framework',
-                      dest='framework',
-                      choices=[frm.value for frm in FrameworkEnum],
-                      help='Specify framework.')
-  parser.add_argument('--fw_version',
-                      dest='fw_version',
-                      type=int,
-                      default=None,
-                      required=False,
-                      help='Specify framework version')
```

Comment:
Would be best to keep these options together as they are related - framework + version